.github/scripts/upload-data.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -euo pipefail
+
+SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+
+share="https://speicherwolke.uni-leipzig.de/index.php/s/MioAzTLMjzbPNyx"
+
+if ! cd_hds_base_url="http://$(docker compose port cd-hds 8080)/fhir"; then
+  >&2 echo "Unable to find clinical domain health data store URL"
+  exit 2
+fi
+
+echo "Uploading consent to gics and patients to cd-hds"
+i=1
+while read -r patient; do
+  patient_id="$(echo "${patient}" | jq -r .key)"
+  "${SCRIPT_DIR}"/upload-consent.sh "${patient_id}" "$(echo "${patient}" | jq -r .value)"
+
+  curl -sf "${share}/download?path=kds&files=${patient_id}.json" \
+  | curl -sf --data-binary @- -H "Content-Type: application/fhir+json" "${cd_hds_base_url}" \
+  >/dev/null
+
+  printf "%s: %8d / %d\n" "${patient_id}" $((i++)) "${1:-10}"
+done < <(curl -sf "${share}/download?files=authored.json" \
+           | jq -c 'to_entries | .[]' | head -n "${1:-10}")
```

Comment:
The script should also check that all consents are available at gICS because it already happened that for some the upload failed.

Reply:
However, this might have been because the gICS wasn't in a clean state.